android/app/src/main/java/org/mozilla/sendandroid/MainActivity.kt

```diff
@@ -35,6 +35,11 @@ class WebAppInterface(private val mContext: MainActivity) {
     fun beginOAuthFlow() {
         mContext.beginOAuthFlow();
     }
+
+    @JavascriptInterface
+    fun shareUrl(url: String) {
+        mContext.shareUrl(url)
```

Comment:
off topic... is there a kotlin linter/formatter we should use?

Reply:
I'll ask the lockbox team.